### PR TITLE
Added Text Utils and Lists component

### PR DESCRIPTION
**Pull request by SaiArunesh**

### CODE REVIEW

**Review by ajitpsakri — APPROVED** [review]

File: components/text-utils/text-utils.css [hunk 1]
```diff
@@ -0,0 +1,50 @@
+h1,
+.xl-text {
+  font-size: 4.8rem;
+  line-height: 6.4rem;
+}
+h2,
```
Comment (ruchiragoswami) [inline components/text-utils/text-utils.css:7]: Just a suggestion, for readability, I like to keep one empty line between code blocks  :)

File: components/text-utils/text-utils.css [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+.large-text {
+  font-size: 3.2rem;
+  line-height: 4rem;
+}
+h3,
```
Comment (ruchiragoswami) [inline components/text-utils/text-utils.css:13]: Just a suggestion, for readability, I like to keep one empty line between code blocks  :)

File: components/text-utils/text-utils.css [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  font-size: 1.6rem;
+  line-height: 2.4rem;
+}
+h5,
+.small-text {
+  font-size: 0.8rem;
+  line-height: 1.6rem;
+}
+
+
```
Comment (ruchiragoswami) [inline components/text-utils/text-utils.css:31]: For keep the code neat, I suggest to remove extra empty line :)

File: components/text-utils/text-utils.css [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+.medium-text {
+  font-size: 2.4rem;
+  line-height: 3.2rem;
+}
+h4,
+.normal-text {
+  font-size: 1.6rem;
+  line-height: 2.4rem;
+}
+h5,
```
Comment (ruchiragoswami) [inline components/text-utils/text-utils.css:25]: Just a suggestion, for readability, I like to keep one empty line between code blocks  :)  This applies to entire file.

Comment (SaiArunesh) [reply components/text-utils/text-utils.css:25]: Fixed em. Thanks

**Pull request merged**